lib/util/ios-proxy.js

```diff
@@ -91,73 +101,82 @@ let iOSProxy = {
     }
   }),
 
-  checkIfRunning() {
+  checkIfRunning: task.async(function*() {
     // Check for any currently running instances of ios_webkit_debug_proxy.
     let running = false;
-    let cmd = "/usr/bin/pgrep";
-    let name = "ios_webkit_debug_proxy";
-    let args = ["-x"];
+    let cmd = "/usr/bin/pgrep -x ";
+    let name = binaryName;
     switch(Runtime.OS) {
       case "Linux": // fall-through
         // "pgrep -x" only matches against the first 15 characters on Linux.
         name = name.slice(0, 15);
       case "Darwin":
-        args.push(name);
+        cmd += name;
+        break;
+      case "WINNT":
+        // "ps" only matches the file name without extension on Windows.
+        name = name.slice(0, -4);
+        cmd = `PowerShell -Command ps ${name}`;
```

Comment:
wmic was a better choice eventually, as tasklist had issues when used in a 64bit system by a 32bit app.